components/src/components/SimpleDateTime/Component.ts

```diff
@@ -59,4 +59,12 @@ export default class Component extends (ParentComponent as any) {
             schema: Component.schema()
         };
     }
+
+    constructor(component) {
+      super(component);
+      // if enableTime is set to false, manually add time fields since it get removed later on through Form IO
+      if (!this.component.enableTime) {
+        this.component.format = this.component.format.concat(' hh:mm a');
+      }
+    }
```

Comment:
Looks like this is causing an exception now? If I drag the component on the setttings bar on hover doesn't come up so it can't be edited. When moving it onto the form it throws this in the console

![image](https://user-images.githubusercontent.com/17445138/121723066-eb83f480-ca9a-11eb-95b3-c49c4b8157fe.png)

It's wierd, one time I tried it it referred to 'component' instead of 'tabs' but not sure where that's coming from

Not happening on any other versions of the app

Reply:
Fixed the constructor by keeping the original arguments for it. Error not showing up on local anymore when using the Simple Date/Time picker.